core/src/main/scala/tech/beshu/ror/utils/yaml/parser.scala

```diff
@@ -89,6 +92,10 @@ object parser {
       case _ => Left(ParsingFailure("Only string keys can be represented in JSON", null))
     }
 
+    def checkDuplicates(jsonObject: JsonObject, key: String): Either[ParsingFailure, String] = {
+      Either.cond(!jsonObject.contains(key), key, ParsingFailure(s"Duplicated key: '$key'", null))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid using null](https://app.codacy.com/app/scarduzio/elasticsearch-readonlyrest-plugin/pullRequest?prid=4570037)

Reply:
@pondzix let's pass there newly created exception with the same message